### DISCUSSION TITLE
Jak wykonać preloader w czystym JavaScript’cie?

### DISCUSSION THREAD

**Discussion by giscus** [robert-orlinski/robertorlinski.pl#17] [Komentarze]
# Jak wykonać preloader w czystym JavaScript’cie?

W tym artykule dowiadujemy się w jaki sposób wykonać prosty preloader w czystym JavaScript'cie, który chowa się po załadowaniu treści strony internetowej.

https://robertorlinski.pl/artykuly/preloader-w-czystym-js

**Comment by Kabuki-Yo** [comment]
No dobrze, ale jak ma się sprawa preloaderów do odbioru przez Google ? Rozumiem, że to ma zatrzymać czytelnika (bo coś się dzieje..,) ale jak algorytmy wyszukiwarek to widzą ?

**Reply by robert-orlinski** [reply]
Google potrafi wykonywać JavaScript, więc "widzi" stronę podobnie jak użytkownik i podobnie ma dostęp do treści, które pokazują się po schowaniu preloadera :)

Inna sprawą jest to czy preloader nie spowalnia w jakiś sposób ładowania strony, bo Google lubi jak najszybsze strony (tak samo jak użytkownicy).
